collections/crowdsecurity/owncloud.yaml

```diff
@@ -0,0 +1,12 @@
+---
+parsers:
+  - crowdsecurity/owncloud-logs
+  - crowdsecurity/owncloud-whitelist
```

Comment:
whitelist doesn't exist anymore replace with `nextcloud` version

Reply:
thank's for your help